back/CinemaReservation.BusinessLayer/Services/AccountService.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Security.Cryptography;
+using Microsoft.AspNetCore.Cryptography.KeyDerivation;
+using CinemaReservation.DataAccessLayer.Entities;
+using CinemaReservation.DataAccessLayer.Contracts;
+using CinemaReservation.BusinessLayer.Models;
+using CinemaReservation.BusinessLayer.Contracts;
+using System.Linq;
+
+namespace CinemaReservation.BusinessLayer.Services
+{
+    public class AccountService: IAccountService
+    {
+        private readonly IUserRepository _userRepository;
+        private readonly ISecurityService _securityService;
+
+        public AccountService(IUserRepository userRepository, ISecurityService securityService)
+        {
+            _userRepository = userRepository;
+            _securityService = securityService;
+        }
+        public async Task<AuthorizationResponseModel> RegisterUser(RegistrationModel registrationModel)
+        {
+            if (_userRepository.GetByEmail(registrationModel.Email) != null)
+            {
+                return null;
+            }
+
+            var isAdmin = false;
```

Comment:
No vars should be used according to my style. If any project that you work allows var then do not mix styles like here